src/prefect/server/schemas/filters.py

```diff
@@ -1180,10 +1180,21 @@ class LogFilterTaskRunId(PrefectFilterBaseModel):
         default=None, description="A list of task run IDs to include"
     )
 
+    is_null_: Optional[bool] = Field(
+        default=None,
+        description="If true, only include logs without a task run id",
+    )
+
     def _get_filter_list(self) -> List:
         filters = []
         if self.any_ is not None:
             filters.append(orm_models.Log.task_run_id.in_(self.any_))
+        if self.is_null_ is not None:
+            filters.append(
+                orm_models.Log.task_run_id == None
```

Comment:
nit: in Python it's best practice to use an `is` to check against `None`; so here and below it'd be `is None` and `is not None`

Reply:
ah jeeze, nice catch, thank you